includes/library/zencart/Services/src/LeadMusicGenreRoutes.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @copyright Copyright 2003-2015 Zen Cart Development Team
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version GIT: $Id: Author: Ian Wilson  Fri Aug 17 17:42:37 2012 +0100 New in v1.5.1 $
+ */
+namespace ZenCart\Services;
+
+/**
+ * Class LeadMusicGenreRoutes
+ * @package ZenCart\Services
+ */
+class LeadMusicGenreRoutes extends LeadRoutes
+{
+    /**
+     * @return bool
+     */
+    public function deleteExecute()
+    {
+        $mainTableFkeyField = $this->listingQuery['mainTable']['fkeyFieldLeft'];
+
+        if ($this->request->readPost('delete_linked') === 'true') {
+            $sql = "SELECT products_id FROM " . TABLE_PRODUCT_MUSIC_EXTRA . " WHERE music_genre_id = :id:";
+            $sql = $this->dbConn->bindVars($sql, ':id:', $this->request->readPost('id'),
+                $this->outputLayout ['fields'] [$mainTableFkeyField] ['bindVarsType']);
```

Comment:
Re code formatting ... Were the spaces between array elements intentional?

... and 2 other other `bindVarsType` places in this same file too ...